src/easylink/steps/rl-dummy/pre-processing/dummy_pre-processing.py

```diff
@@ -25,22 +27,30 @@ def load_file(file_path, file_format=None):
 
 # DATASET_FILE_PATHS is list of filepaths
 dataset_paths = os.environ["DATASET_FILE_PATHS"].split(",")
+logging.info(f"dataset_paths={dataset_paths}")
```

Comment:
This is totally fine as-is but here's a cool f-string trick you might not know about: this is equivalent to what you have
```suggestion
logging.info(f"{dataset_paths=}")
```